<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Non-Precision Final Approach</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>529</width>
         <height>576</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="inputLayersGroupBox">
          <property name="title">
           <string>Input Layers</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="thresholdLayerLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Threshold Point</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="navaidLayerLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>NAVAID Point</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="runwayLayerLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Runway Line</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QgsMapLayerComboBox" name="runwayLayerComboBox"/>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QgsMapLayerComboBox" name="thresholdLayerComboBox"/>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QgsMapLayerComboBox" name="navaidLayerComboBox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calculateInterceptpushButton">
          <property name="text">
           <string>Calculate Track Offset for 1400m Centreline Intercept</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="inputParametersGroupBox">
          <property name="title">
           <string>Input Parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="7" column="1">
            <widget class="QDoubleSpinBox" name="segmentLengthSpinBox">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="maximum">
              <double>100000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.000100000000000</double>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::DefaultStepType</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="navaidTypeLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>NAVAID Type</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="segmentLengthLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Segment Length</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="thresholdElevationSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>-3000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>30000.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.000100000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Threshold Elevation</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QComboBox" name="thresholdElevationUnitsComboBox"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="trackOffsetLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Track Offset Angle</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="3">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="6" column="1">
            <widget class="QDoubleSpinBox" name="trackOffsetSpinBox">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="minimum">
              <double>-30.000000000000000</double>
             </property>
             <property name="maximum">
              <double>30.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.000100000000000</double>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QComboBox" name="navaidTypeComboBox">
             <item>
              <property name="text">
               <string>NDB</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>VOR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LOC</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>DF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PSR</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QComboBox" name="distanceUnitsComboBox"/>
           </item>
           <item row="6" column="2">
            <widget class="QComboBox" name="trackOffsetUnitsComboBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Final Approach Track Direction</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QComboBox" name="directionFATComboBox">
             <item>
              <property name="text">
               <string>TO NAVAID</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FROM NAVAID</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QCheckBox" name="drawTrackToleranceCheckBox">
             <property name="text">
              <string>    Draw Tracking Tolerance</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="outputGroupBox">
          <property name="title">
           <string>Output</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="exportKmlCheckBox">
             <property name="text">
              <string>Export to KML</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="outputFolderLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calculateButton">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="logGroupBox">
          <property name="title">
           <string>Log</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTextEdit" name="logTextEdit"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
